Update homepage content (#99)
acid-info/waku.org: src/pages/index.mdx

    </HeroActions>
  </HeroInfo>
  <HeroAsset>
    <video autoPlay loop muted playsInline>
      <source src="/hero/hero-video.mov" type='video/mp4; codecs="hvc1"' />
      <source src="/hero/hero-video.webm" type="video/webm" />
    </video>
  </HeroAsset>
</Hero>

<Box top={{ xs: 144, lg: 190 }}>
  <CallToActionSection
    title="Scalable, Reliable, and Secure Decentralized Communication"
    columns={1}
    border={false}
    align="center"
    description={<>

>       Waku delivers a unified technology stack addressing the unique challenges of decentralized peer-to-peer communication, offering turnkey reliability, scalability, and censorship resistance for your applications. Built on libp2p, Waku enhances its capabilities to ensure optimal performance in diverse network conditions.<br /><br />

Reviewer: This is not what I meant when I said I wanted an updated based on the explanation series.

What I want, is a writer to ingest the explanation series content, and revisit the home page, so that what is highlight in explanation series is present from the moment you open the page.

Which means reworkign the "Features" section and potentially the title and intro of the page.

    timeline={[
      {
        period: [2013],
        description: "Ethereum's Whisper unveiled as web3 messaging protocol."
      },
      {
        period: [2018],
        description: "Status establishes Vac to R&D Whisper improvements."
      },
      {
        period: [2020],
        description: "Waku v1 replaces Whisper in Status app; Waku v2 with Relay PoC and spec with relay."
      },
      {
        period: [2021],
        description: "Waku v2 specs and PoC released with RLN Relay and light protocols; JS-Waku (for browsers) and Go-Waku released."
      },
      {
        period: [2022],
        description: "Discv5 and DNS Discovery added; SQLite for Waku Store; RLN Relay testnet debuts at Devcon 6."
      },
      {
        period: [2023],
        description: "Relay sharding, including auto-sharding; Launch of The Waku Network, a permissionless proto-network DoS protected by RLNv1."
      },
      {
        period: [2024, "H1"],

>         description: "Functional Waku edge node in Status mobile, using light protocols; store v3 protocol to enable hash message queries, allowing implementation of peer-to-peer reliabilty protocols."

Reviewer: ```suggestion
        description: "Functional Waku edge node in Status mobile; store v3 protocol enables message has queries, allowing implementation of reliability protocols."
```

    timeline={[
      {
        period: [2013],
        description: "Ethereum's Whisper unveiled as web3 messaging protocol."
      },
      {
        period: [2018],
        description: "Status establishes Vac to R&D Whisper improvements."
      },
      {
        period: [2020],
        description: "Waku v1 replaces Whisper in Status app; Waku v2 with Relay PoC and spec with relay."
      },
      {
        period: [2021],
        description: "Waku v2 specs and PoC released with RLN Relay and light protocols; JS-Waku (for browsers) and Go-Waku released."
      },
      {
        period: [2022],
        description: "Discv5 and DNS Discovery added; SQLite for Waku Store; RLN Relay testnet debuts at Devcon 6."
      },
      {
        period: [2023],

>         description: "Relay sharding, including auto-sharding; opt-in spam-protected topic; Launch of The Waku Network, a permissionless proto-network DoS protected by RLNv1."

Reviewer: you can remove "opt-in spam-protected topic"

      },
      {
        period: [2024, "H1"],
        description: "Functional Waku edge node in Status mobile, using light protocols; store v3 protocol to enable hash message queries, allowing implementation of peer-to-peer reliabilty protocols."
      },
      {
        period: [2024, "H2"],

>         description: "RLNv2: Flexible rate limit, onchain Merkle tree for faster initialisation, definition of membership economics. Tunable peer-to-peer reliability protocols; Scalable Data Sync protocol: end-to-end reliability protocol; Hardening of Status chat protocols."

Reviewer: ```suggestion
        description: "RLNv2: Flexible rate limit & onchain Merkle tree. Peer-to-peer and end-to-end reliability protocols."
```

Looked too verbose

      },
      {
        period: [2025, "+"],

>         description: "Simplified Waku Message API across all implementations. Incentivization, RLN in mainnet and browser, store decentralization. Status chat protocols scalability and optimization."

Reviewer: ```suggestion
        description: "Simplify Waku API. Incentivization, RLN in mainnet and browser, Status chat protocols scalability and optimization."
```
